sources/documents/RTB/NormeDiProgetto/Supporto/documentazione.tex

```diff
@@ -0,0 +1,152 @@
+\subsection{Documentazione}
+
+\subsubsection{Descrizione}
+Il processo di documentazione registra l'informazione generata da altri processi o attività. Il processo contiene l'insieme di attività che pianificano, producono, modificano, rilasciano e mantengono i documenti legati al progetto.\\
+Il processo consiste nelle seguenti attività:
+\begin{itemize}
+  \item Implementazione del processo;
+  \item Progettazione e sviluppo;
+  \item Rilascio.
+\end{itemize}
+
+\paragraph{Implementazione del processo}\label{implementazioneprocessodocumentazione}
+Questa attività definisce quali documenti saranno generati durante il progetto, definendo per ciascuno:
+\begin{itemize}
+  \item Titolo;
+  \item Scopo;
+  \item Descrizione;
+  \item Responsabilità per input, redazione, verifica e approvazione;
```

Comment:
"input" in che senso?

Reply:
inteso come "fornire contributo", ad esempio nei casi in cui non è il solo redattore ad essere in possesso delle informazioni necessarie. Sostituirò con "contribuzione" per essere più chiaro